### PR TITLE
Turning in web app toolbox

### PR DESCRIPTION
**Pull request by segerpeter07**

### CODE REVIEW

File: templates/profile.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>
+            Redirect
+        </title>
+    </head>
+<body>
+    <h1>Welcome Patrick Huston!</h1>
```
Comment (segerphilip) [inline templates/profile.html:9]: Probably don't want to hardcode this? When I log in as me, I become Patrick.

File: hello.py [hunk 2]
```diff
@@ -3,11 +3,39 @@
...
-@app.route('/')
+
+@app.route('/', methods=['GET', 'POST'])
 def hello_world():
-    return 'Hello World!'
+    return render_template('index.html')
+
+
+@app.route('/hello/', methods=['GET', 'POST'])
+@app.route('/hello/<name>')
```
Comment (segerphilip) [inline hello.py:17]: Manually calling this with my name renders it as expected.

**Review by segerphilip — COMMENTED** [review]
Definitely could use some work in the style department, but works as expected (besides everyone being Patrick) 👍 Nicely done.

File: templates/profile.html (hunk 1 above)
Comment (segerpeter07) [reply templates/profile.html:9]: That's part of the assignment...

Comment (segerphilip) [reply templates/profile.html:9]: screen shot 2017-02-13 at 1 51 32 pm 🤣

**Pull request closed without merging**